apps/src/templates/teacherDashboard/AssignmentSelector.jsx

```diff
@@ -337,6 +348,7 @@ export default class AssignmentSelector extends Component {
                     </option>
                   )
               )}
+              <option value={noAssignment} />
```

Comment:
why is this at the end now? I think we would want to have a good reason to change this if it affects the ordering in the UI.

Reply:
Oh, Amanda asked me to move it to make it slightly less likely that folks will pick this option.